src/main/java/com/synopsys/integration/alert/web/api/job/JobConfigController.java

```diff
@@ -100,6 +100,7 @@ public ValidationResponseModel validate(JobFieldModel requestBody) {
 
     @Override
     public ValidationResponseModel test(JobFieldModel resource) {
-        return ResponseFactory.createContentResponseFromAction(jobConfigActions.test(resource));
+        ValidationActionResponse response = jobConfigActions.test(resource);
+        return ResponseFactory.createContentResponseFromAction(new ValidationActionResponse(HttpStatus.OK, response.getContent().orElse(null)));
```

Comment:
If we're ok returning null, then why do we work with an optional as Content?

Reply:
I brought this same issue up in the video code review. It shouldn't ever actually return null, but we should have a static initializer for validation responses anyway.

Reply:
ActionResponse has the getContent() method and in some cases there is no content to return it can be null you just want the http status code and/or message with no content. So because content can be null we return an Optional to indicate to other users of the API that the content can be null and avoid NullPointerExceptions.  The Optional allows you to know you need to check if it is present otherwise you would need to have special code to check for null otherwise you would have NullPointerExceptions.  ValidationActionResponse extends ActionResponse. Because we are wrapping the ValidationActionResponse with a new ValidationActionResponse with an OK status to represent validation or test execution, we need to get the content with getContent() which is an Optional.  So to work with the Optional API i have the orElse(null) for the empty optional case.

It's a little beyond the scope of this PR but we might be able to refactor the API to make it a clearer if there's confusion.